File: Medical_Profile/storage.vb
Partial Class form1
 'Microsoft Sans Serif
 ' Public scr_font As System.Drawing.Font = New Font(10.25,
 Public scr_font As System.Drawing.Font = New Font("Calibri", 10.25, FontStyle.Regular)

 Public scb_font As System.Drawing.Font = New Font("Calibri", 10.25, FontStyle.Bold)

 Public pidep As ErrorProvider = New ErrorProvider

 Public patep As ErrorProvider = New ErrorProvider

 Public inpid As Boolean = False

 Public inpat As Boolean = False

 Public firstname As String = ""

 Public lastname As String = ""

 Public installation_key As String = ""

 Public installation_url As String = ""

 Public valid_patientid As Boolean = True

 Public valid_patient As Boolean = True

 Public log As Boolean = False

 Public pname As String

 Public pdob As String

 Public rdoc As String

 Public logfile As String

 Public valfld As String

 Public fullname As String = String.Empty

 Public read_name As String = String.Empty

 Public read_path As String = String.Empty

 Public selected_name As String = String.Empty

 Public lines_setting As String

 Public label_length As Single = 72.0 * 2.7

 Public total_lines As Integer

 Public labels_number As Integer

 Public lines_number As Integer

 Public blocks_number As Integer

 Public minimum_blocks As Integer

 Public Screen_width As Integer

 Public Screen_height As Integer

 Public redo_bn As Integer

 Public eval_enc As String = Nothing

 Public Patient_limit As Integer

 Private _Dalt As Boolean = False

 Public USB_Fname As String = Nothing

 Public USB_Ftime As DateTimeOffset = Nothing

 Public aws_body As Dictionary(Of String, Object) = New Dictionary(Of String, Object)

 Public saved_patients As SortedDictionary(Of String, Dsave) = New SortedDictionary(Of String, Dsave)

 Public dict_dsaves As Dictionary(Of String, String) = New Dictionary(Of String, String)

 Public read_endpoint_file As Boolean = False

 Public loading As Boolean = False

 Public rethit As Boolean = False

 Public tabhit As Boolean = False

 Public Window_Text As String = Nothing

 Public file_clear As Boolean = False

 Public allow_paint As Boolean = True

 Public max_lines_flag As Boolean = True

 Public Pnglablist As List(Of Byte()) = New List(Of Byte())

 Delegate Sub Setfl(ByVal source As Object, ByVal e As System.IO.FileSystemEventArgs)

 Public blocks As Dictionary(Of Integer, Blk_info) = New Dictionary(Of Integer, Blk_info)

 Public bfont As Dictionary(Of String, Boolean) = New Dictionary(Of String, Boolean)

 Public mp_backcolor As System.Drawing.Color = System.Drawing.Color.FromArgb(126, 255, 255)

 Public drive_label_encoded As String = Nothing

 Public eval_encoded As String = Nothing

 Public file_access As Boolean = False

 Public testmode As Boolean = False

 Public save_ver As Integer = 1

 Public run_timer As Boolean = False

 Private bl_loaded As SortedDictionary(Of String, Blk_entry) = New SortedDictionary(Of String, Blk_entry)

 Public bl_available As SortedDictionary(Of Integer, String) = New SortedDictionary(Of Integer, String)

 Public bl_used As SortedDictionary(Of Integer, String) = New SortedDictionary(Of Integer, String)

 Public maxlines As Dictionary(Of String, Integer) = New Dictionary(Of String, Integer)

 Public patientid_val As String = Nothing

 Public first_run As Boolean = True

 Public originaly As Integer

 Public currenty As Integer

 Public SW As Stopwatch = New Stopwatch()

 Public title As String = "This program is named medical profile"

 Public fieldsgb As List(Of String) = New List(Of String)

 Public fieldsmr As Dictionary(Of String, Integer) = New Dictionary(Of String, Integer)

 Public lab1 As Dictionary(Of String, String) = New Dictionary(Of String, String)

 Public labgb As Dictionary(Of String, String) = New Dictionary(Of String, String)

 Public max_rec As Dictionary(Of Integer, Integer) = New Dictionary(Of Integer, Integer)

 Public f3s As Dictionary(Of String, Form3) = New Dictionary(Of String, Form3)

 Public saved_type As Boolean = False

 Public Dsi As KeyValuePair(Of String, Dsave) = New KeyValuePair(Of String, Dsave)

 Public Ds As Dsave = Nothing

 Public dsn As String

 Public usbd As Dictionary(Of String, String) = New Dictionary(Of String, String)()

 Public ath_blist As List(Of Ath_block) = New List(Of Ath_block)

 Public gpn As Dictionary(Of Integer, String) = New Dictionary(Of Integer, String)
 Public gph As Dictionary(Of Integer, String) = New Dictionary(Of Integer, String)
 Public gpb As Dictionary(Of Integer, String) = New Dictionary(Of Integer, String)

 Public athp As Dictionary(Of String, String) = New Dictionary(Of String, String)
 Public atdp As Dictionary(Of String, Integer) = New Dictionary(Of String, Integer)

 Public departments As Dictionary(Of Integer, Dept_Return) = New Dictionary(Of Integer, Dept_Return)

 Public atpv As Dictionary(Of Integer, String) = New Dictionary(Of Integer, String)

 Public providers As Dictionary(Of Integer, Provider_return) = New Dictionary(Of Integer, Provider_return)

 Dim L1_ret As Level1_Return = Nothing

 Dim L2_ret As Level2_Return = Nothing

 Public Shared installed_version As String = Nothing

 Private emppn As Panel = Nothing

 Dim Person_loaded As Boolean = False

 Public cr As String = Chr(13)
 Public lftab As String = Chr(10) & Chr(9)
 Public lf As String = Chr(10)
 Public crlf As String = Chr(13) & Chr(10)
 Public tab As String = Chr(9)

 Public cwd As Single() = {0.0, 0.0, 0.226, 0.579, 0.579, 0.579, 0.579, 0.579, 0.579, 0.579, 0.579, 0.579, 0.579, 0.0, 0.226,
      0.579, 0.579, 0.579, 0.579, 0.579, 0.579, 0.579, 0.579, 0.579, 0.579, 0.579, 0.763, 0.763, 0.544, 0.533,
      0.533, 0.533, 0.226, 0.326, 0.401, 0.498, 0.507, 0.715, 0.682, 0.221, 0.303, 0.303, 0.498, 0.498, 0.25,
      0.306, 0.252, 0.386, 0.507, 0.507, 0.507, 0.507, 0.507, 0.507, 0.507, 0.507, 0.507, 0.507, 0.268, 0.268,
      0.498, 0.498, 0.498, 0.463, 0.894, 0.579, 0.544, 0.533, 0.615, 0.488, 0.459, 0.631, 0.623, 0.252, 0.319,
      0.52, 0.42, 0.855, 0.646, 0.662, 0.517, 0.673, 0.543, 0.459, 0.487, 0.642, 0.567, 0.89, 0.519, 0.487,
      0.468, 0.307, 0.386, 0.307, 0.498, 0.498, 0.291, 0.479, 0.525, 0.423, 0.525, 0.498, 0.305, 0.471, 0.525,
      0.229, 0.239, 0.455, 0.229, 0.799, 0.525, 0.527, 0.525, 0.525, 0.349, 0.391, 0.335, 0.525, 0.452, 0.715,
      0.433, 0.453, 0.395, 0.314, 0.46, 0.314, 0.498, 0.226, 0.326, 0.498, 0.507, 0.498, 0.507, 0.498, 0.498,
      0.393, 0.834, 0.402, 0.512, 0.498, 0.306, 0.507, 0.394, 0.339, 0.498, 0.336, 0.334, 0.292, 0.55, 0.586,
      0.252, 0.307, 0.246, 0.422, 0.512, 0.636, 0.671, 0.675, 0.463, 0.579, 0.226, 0.326, 0.498, 0.507, 0.498,
      0.507, 0.498, 0.498, 0.393, 0.834, 0.402, 0.512, 0.498, 0.306, 0.507, 0.394, 0.339, 0.498, 0.336, 0.334,
      0.292, 0.55, 0.586, 0.252, 0.307, 0.246, 0.422, 0.512, 0.636, 0.671, 0.675, 0.463, 0.579, 0.579, 0.579,
      0.579, 0.579, 0.579, 0.763, 0.533, 0.488, 0.488, 0.488, 0.488, 0.252, 0.252, 0.252, 0.252, 0.625, 0.646,
      0.662, 0.662, 0.662, 0.662, 0.662, 0.498, 0.664, 0.642, 0.642, 0.642, 0.642, 0.487, 0.517, 0.527, 0.479,
      0.479, 0.479, 0.479, 0.479, 0.479, 0.773, 0.423, 0.498, 0.498, 0.498, 0.498, 0.229, 0.229, 0.229, 0.229,
      0.525, 0.525, 0.527, 0.527, 0.527, 0.527, 0.527, 0.498, 0.529, 0.525, 0.525, 0.525, 0.525, 0.453, 0.525,
      0.453}

 Public cwdb As Single() = {0.0, 0.0, 0.226, 0.606, 0.606, 0.606, 0.606, 0.606, 0.606, 0.606, 0.606, 0.606, 0.606, 0.0, 0.226,
      0.606, 0.606, 0.606, 0.606, 0.606, 0.606, 0.606, 0.606, 0.606, 0.606, 0.606, 0.775, 0.775, 0.561, 0.529,
      0.529, 0.529, 0.226, 0.326, 0.438, 0.498, 0.507, 0.729, 0.705, 0.233, 0.312, 0.312, 0.498, 0.498, 0.258,
      0.306, 0.267, 0.43, 0.507, 0.507, 0.507, 0.507, 0.507, 0.507, 0.507, 0.507, 0.507, 0.507, 0.276, 0.276,
      0.498, 0.498, 0.498, 0.463, 0.898, 0.606, 0.561, 0.529, 0.63, 0.488, 0.459, 0.637, 0.631, 0.267, 0.331,
      0.547, 0.423, 0.874, 0.659, 0.676, 0.532, 0.686, 0.563, 0.473, 0.495, 0.653, 0.591, 0.906, 0.551, 0.52,
      0.478, 0.325, 0.43, 0.325, 0.498, 0.498, 0.3, 0.494, 0.537, 0.418, 0.537, 0.503, 0.316, 0.474, 0.537,
      0.246, 0.255, 0.48, 0.246, 0.813, 0.537, 0.538, 0.537, 0.537, 0.355, 0.399, 0.347, 0.537, 0.473, 0.745,
      0.459, 0.474, 0.397, 0.344, 0.475, 0.344, 0.498, 0.226, 0.326, 0.498, 0.507, 0.498, 0.507, 0.498, 0.498,
      0.415, 0.834, 0.416, 0.539, 0.498, 0.306, 0.507, 0.39, 0.342, 0.498, 0.338, 0.336, 0.301, 0.563, 0.598,
      0.268, 0.303, 0.252, 0.435, 0.539, 0.658, 0.691, 0.702, 0.463, 0.606, 0.226, 0.326, 0.498, 0.507, 0.498,
      0.507, 0.498, 0.498, 0.415, 0.834, 0.416, 0.539, 0.498, 0.306, 0.507, 0.39, 0.342, 0.498, 0.338, 0.336,
      0.301, 0.563, 0.598, 0.268, 0.303, 0.252, 0.435, 0.539, 0.658, 0.691, 0.702, 0.463, 0.606, 0.606, 0.606,
      0.606, 0.606, 0.606, 0.775, 0.529, 0.488, 0.488, 0.488, 0.488, 0.267, 0.267, 0.267, 0.267, 0.639, 0.659,
      0.676, 0.676, 0.676, 0.676, 0.676, 0.498, 0.681, 0.653, 0.653, 0.653, 0.653, 0.52, 0.532, 0.555, 0.494,
      0.494, 0.494, 0.494, 0.494, 0.494, 0.775, 0.418, 0.503, 0.503, 0.503, 0.503, 0.246, 0.246, 0.246, 0.246,
      0.537, 0.537, 0.538, 0.538, 0.538, 0.538, 0.538, 0.498, 0.544, 0.537, 0.537, 0.537, 0.537, 0.474, 0.537,
      0.474
     }

 Public Property Emppn1 As Panel
  Get
   Return emppn
  End Get
  Set(value As Panel)
   emppn = value
  End Set
 End Property

 Public Property Data_altered As Boolean
  Get
   Return _Dalt
  End Get
  Set(value As Boolean)
   _Dalt = value
  End Set
 End Property

End Class

File: Medical_Profile/storage.cs
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using MPClabel;
using Enc256;
namespace Medical_Profile
{
 public partial class Form1
 {
  public MPClabel.ILabint Dlab = LabFactory.GetLab();
  public Ienc256 Ede = EncFactory.GetEnc();
  public Font scr_font = new Font("Calibri", 10.25F, FontStyle.Regular);
  public Font scb_font = new Font("Calibri", 10.25F, FontStyle.Bold);
  public ErrorProvider pidep = new ErrorProvider();
  public ErrorProvider patep = new ErrorProvider();
  public bool inpid = false;
  public bool inpat = false;
  public string firstname = "";
  public string lastname = "";
  public string installation_key = "";
  public string installation_url = "";
  public bool valid_patientid = true;
  public bool valid_patient = true;
  public bool log = false;
  public string pname;
  public string pdob;
  public string rdoc;
  public string logfile;
  public string valfld;
  public string fullname = string.Empty;
  public string read_name = string.Empty;
  public string read_path = string.Empty;
  public string selected_name = string.Empty;
  public string lines_setting;
  public float label_length = (float)(72.0 * 2.7);
  public int total_lines;
  public int labels_number;
  public int lines_number;
  public int blocks_number;
  public int minimum_blocks;
  public int Screen_width;
  public int Screen_height;
  public System.Drawing.Size Form_size;
  public System.Drawing.Size Form_area;

  public Panel Pn;
  public Panel Pe;
  public int Ystart = 0;
  public int Original_Margin = -1;
  public int W_delta;
  public int _H_delta;
  public Rectangle Form_Bounds;
  public Rectangle Screen_Area;

  public int redo_bn;
  public string eval_enc = null;
  public int Patient_limit;
  private bool _Dalt = false;
  public string USB_Fname = null;
  public DateTimeOffset USB_Ftime = default;
  public Dictionary<string, object> aws_body = new Dictionary<string, object>();
  public SortedDictionary<string, Dsave> saved_patients = new SortedDictionary<string, Dsave>();
  public Dictionary<string, string> dict_dsaves = new Dictionary<string, string>();
  public bool read_endpoint_file = false;
  public bool loading = false;
  public bool rethit = false;
  public bool tabhit = false;
  public string Window_Text = null;
  public bool file_clear = false;
  public bool allow_paint = true;
  public bool max_lines_flag = true;
  public List<byte[]> Pnglablist = new List<byte[]>();

  public bool Vsm = false;
  public VScrollBar Sb = new VScrollBar();
  public int Vertical_Scroll_Position = 0;
  public int Form_Height = -1;

  public delegate void Setfl(object source, System.IO.FileSystemEventArgs e);

  public Dictionary<int, Blk_info> blocks = new Dictionary<int, Blk_info>();
  public Dictionary<string, bool> bfont = new Dictionary<string, bool>();
  public System.Drawing.Color mp_backcolor = System.Drawing.Color.FromArgb(126, 255, 255);
  public string drive_label_encoded = null;
  public string eval_encoded = null;
  public bool file_access = false;
  public bool testmode = false;
  public int save_ver = 1;
  public bool run_timer = false;
  private SortedDictionary<string, Blk_entry> bl_loaded = new SortedDictionary<string, Blk_entry>();
  public SortedDictionary<int, string> bl_available = new SortedDictionary<int, string>();
  public SortedDictionary<int, string> bl_used = new SortedDictionary<int, string>();
  public Dictionary<string, int> maxlines = new Dictionary<string, int>();
  public string patientid_val = null;
  public bool first_run = true;
  public int Originaly;
  public int currenty;
  public Stopwatch SW = new Stopwatch();
  public string title = "This program is named medical profile";
  public List<string> fieldsgb = new List<string>();
  public Dictionary<string, int> fieldsmr = new Dictionary<string, int>();
  public Dictionary<string, string> lab1 = new Dictionary<string, string>();
  public Dictionary<string, string> labgb = new Dictionary<string, string>();
  public Dictionary<int, int> Max_rec = new Dictionary<int, int>();
  public Dictionary<string, Form3> f3s = new Dictionary<string, Form3>();
  public bool saved_type = false;
  public KeyValuePair<string, Dsave> Dsi = new KeyValuePair<string, Dsave>();
  public Dsave Ds = null;
  public string dsn;
  public Dictionary<string, string> usbd = new Dictionary<string, string>();
  public List<Ath_block> ath_blist = new List<Ath_block>();
  public Dictionary<int, string> gpn = new Dictionary<int, string>();
  public Dictionary<int, string> gph = new Dictionary<int, string>();
  public Dictionary<int, string> gpb = new Dictionary<int, string>();
  public Dictionary<int, GroupBox> Grpblocks = new Dictionary<int, GroupBox>();
  public Dictionary<string, string> athp = new Dictionary<string, string>();
  public Dictionary<string, int> atdp = new Dictionary<string, int>();
  public Dictionary<int, Dept_Return> departments = new Dictionary<int, Dept_Return>();
  public Dictionary<int, string> atpv = new Dictionary<int, string>();
  public Dictionary<int, Provider_return> providers = new Dictionary<int, Provider_return>();
  private Level1_Return L1_ret = null;
  private Level2_Return L2_ret = null;
  public static string installed_version = null;
  private Panel emppn = null;
//  private bool Person_loaded = false;
  public string cr =Convert.ToString((char)13);
  public string lftab = Convert.ToString((char)10) + Convert.ToString((char)9);
  public string lf = Convert.ToString((char)10);
  public string crlf = Convert.ToString((char)13) + Convert.ToString((char)10);
  private string tab = Convert.ToString((char)9);
  private float[] cwd = new[] { 0.0F, 0.0F, 0.226F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.0F, 0.226F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.763F, 0.763F, 0.544F, 0.533F, 0.533F, 0.533F, 0.226F, 0.326F, 0.401F, 0.498F, 0.507F, 0.715F, 0.682F, 0.221F, 0.303F, 0.303F, 0.498F, 0.498F, 0.25F, 0.306F, 0.252F, 0.386F, 0.507F, 0.507F, 0.507F, 0.507F, 0.507F, 0.507F, 0.507F, 0.507F, 0.507F, 0.507F, 0.268F, 0.268F, 0.498F, 0.498F, 0.498F, 0.463F, 0.894F, 0.579F, 0.544F, 0.533F, 0.615F, 0.488F, 0.459F, 0.631F, 0.623F, 0.252F, 0.319F, 0.52F, 0.42F, 0.855F, 0.646F, 0.662F, 0.517F, 0.673F, 0.543F, 0.459F, 0.487F, 0.642F, 0.567F, 0.89F, 0.519F, 0.487F, 0.468F, 0.307F, 0.386F, 0.307F, 0.498F, 0.498F, 0.291F, 0.479F, 0.525F, 0.423F, 0.525F, 0.498F, 0.305F, 0.471F, 0.525F, 0.229F, 0.239F, 0.455F, 0.229F, 0.799F, 0.525F, 0.527F, 0.525F, 0.525F, 0.349F, 0.391F, 0.335F, 0.525F, 0.452F, 0.715F, 0.433F, 0.453F, 0.395F, 0.314F, 0.46F, 0.314F, 0.498F, 0.226F, 0.326F, 0.498F, 0.507F, 0.498F, 0.507F, 0.498F, 0.498F, 0.393F, 0.834F, 0.402F, 0.512F, 0.498F, 0.306F, 0.507F, 0.394F, 0.339F, 0.498F, 0.336F, 0.334F, 0.292F, 0.55F, 0.586F, 0.252F, 0.307F, 0.246F, 0.422F, 0.512F, 0.636F, 0.671F, 0.675F, 0.463F, 0.579F, 0.226F, 0.326F, 0.498F, 0.507F, 0.498F, 0.507F, 0.498F, 0.498F, 0.393F, 0.834F, 0.402F, 0.512F, 0.498F, 0.306F, 0.507F, 0.394F, 0.339F, 0.498F, 0.336F, 0.334F, 0.292F, 0.55F, 0.586F, 0.252F, 0.307F, 0.246F, 0.422F, 0.512F, 0.636F, 0.671F, 0.675F, 0.463F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.579F, 0.763F, 0.533F, 0.488F, 0.488F, 0.488F, 0.488F, 0.252F, 0.252F, 0.252F, 0.252F, 0.625F, 0.646F, 0.662F, 0.662F, 0.662F, 0.662F, 0.662F, 0.498F, 0.664F, 0.642F, 0.642F, 0.642F, 0.642F, 0.487F, 0.517F, 0.527F, 0.479F, 0.479F, 0.479F, 0.479F, 0.479F, 0.479F, 0.773F, 0.423F, 0.498F, 0.498F, 0.498F, 0.498F, 0.229F, 0.229F, 0.229F, 0.229F, 0.525F, 0.525F, 0.527F, 0.527F, 0.527F, 0.527F, 0.527F, 0.498F, 0.529F, 0.525F, 0.525F, 0.525F, 0.525F, 0.453F, 0.525F, 0.453F };
  private float[] cwdb = new[] { 0.0F, 0.0F, 0.226F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.0F, 0.226F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.775F, 0.775F, 0.561F, 0.529F, 0.529F, 0.529F, 0.226F, 0.326F, 0.438F, 0.498F, 0.507F, 0.729F, 0.705F, 0.233F, 0.312F, 0.312F, 0.498F, 0.498F, 0.258F, 0.306F, 0.267F, 0.43F, 0.507F, 0.507F, 0.507F, 0.507F, 0.507F, 0.507F, 0.507F, 0.507F, 0.507F, 0.507F, 0.276F, 0.276F, 0.498F, 0.498F, 0.498F, 0.463F, 0.898F, 0.606F, 0.561F, 0.529F, 0.63F, 0.488F, 0.459F, 0.637F, 0.631F, 0.267F, 0.331F, 0.547F, 0.423F, 0.874F, 0.659F, 0.676F, 0.532F, 0.686F, 0.563F, 0.473F, 0.495F, 0.653F, 0.591F, 0.906F, 0.551F, 0.52F, 0.478F, 0.325F, 0.43F, 0.325F, 0.498F, 0.498F, 0.3F, 0.494F, 0.537F, 0.418F, 0.537F, 0.503F, 0.316F, 0.474F, 0.537F, 0.246F, 0.255F, 0.48F, 0.246F, 0.813F, 0.537F, 0.538F, 0.537F, 0.537F, 0.355F, 0.399F, 0.347F, 0.537F, 0.473F, 0.745F, 0.459F, 0.474F, 0.397F, 0.344F, 0.475F, 0.344F, 0.498F, 0.226F, 0.326F, 0.498F, 0.507F, 0.498F, 0.507F, 0.498F, 0.498F, 0.415F, 0.834F, 0.416F, 0.539F, 0.498F, 0.306F, 0.507F, 0.39F, 0.342F, 0.498F, 0.338F, 0.336F, 0.301F, 0.563F, 0.598F, 0.268F, 0.303F, 0.252F, 0.435F, 0.539F, 0.658F, 0.691F, 0.702F, 0.463F, 0.606F, 0.226F, 0.326F, 0.498F, 0.507F, 0.498F, 0.507F, 0.498F, 0.498F, 0.415F, 0.834F, 0.416F, 0.539F, 0.498F, 0.306F, 0.507F, 0.39F, 0.342F, 0.498F, 0.338F, 0.336F, 0.301F, 0.563F, 0.598F, 0.268F, 0.303F, 0.252F, 0.435F, 0.539F, 0.658F, 0.691F, 0.702F, 0.463F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.606F, 0.775F, 0.529F, 0.488F, 0.488F, 0.488F, 0.488F, 0.267F, 0.267F, 0.267F, 0.267F, 0.639F, 0.659F, 0.676F, 0.676F, 0.676F, 0.676F, 0.676F, 0.498F, 0.681F, 0.653F, 0.653F, 0.653F, 0.653F, 0.52F, 0.532F, 0.555F, 0.494F, 0.494F, 0.494F, 0.494F, 0.494F, 0.494F, 0.775F, 0.418F, 0.503F, 0.503F, 0.503F, 0.503F, 0.246F, 0.246F, 0.246F, 0.246F, 0.537F, 0.537F, 0.538F, 0.538F, 0.538F, 0.538F, 0.538F, 0.498F, 0.544F, 0.537F, 0.537F, 0.537F, 0.537F, 0.474F, 0.537F, 0.474F };

  private Panel Emppn1
  {
   get
   {
    return emppn;
   }

   set
   {
    emppn = value;
   }
  }

  public bool Data_altered
  {
   get
   {
    return _Dalt;
   }

   set
   {
    _Dalt = value;
   }
  }
  public int H_delta
  {
   get
   {
    return _H_delta;
   }

   set
   {
    _H_delta = value;
   }
  }
 }
}